docs/packages/components/Checkbox/index.js

```diff
@@ -26,17 +33,23 @@ export const Checkbox = ({
   name,
   value,
   override,
+  isReversed,
 }) => {
   const Override = {
     wrapper: {
+      textAlign: 'center',
       lineHeight: 'checkbox',
+      display: 'inline-flex',
+      flexDirection: isReversed ? 'row-reverse' : 'row',
+      justifyContent: isReversed ? 'flex-end' : 'start',
       cursor: isDisabled ? 'not-allowed' : 'pointer',
-      ...getAllowedProps(override.wrapper, ['typography']),
+      ...getAllowedProps(override.wrapper, [...layout, ...typography, ...flex]),
```

Comment:
I don't think layout is required here. Should wrap this with a box.

Reply:
These changes where done by Govind, I haven't looked very closely on the props. My concentration was to bring the docz up properly for all component.
But here Layout is not required for checkbox.
But we need the space

Reply:
@navin-moorthy The build for this branch breaks. Is it this branch where the docz are working for Text and Heading component ?